analyses/country_template/docs/trigger_report_template.html

```diff
@@ -0,0 +1,622 @@
+<!DOCTYPE html>
```

Comment:
If this is generated, then maybe it doesn't need to be checked in?

Reply:
I figured this way it is easy to share with outside, non-technical folks. But I guess this is the template so we wouldn't share an "empty" template. I'll remove it.